### ISSUE TITLE
Allocate 3 FC430s and 3FC830s for an OpenShift Beta cluster

### ISSUE THREAD

**Issue by joachimweyl** [nerc-project/operations#549]
## Motivation
RH requires an OpenShift Beta Cluster for testing the Beta builds of OpenShift.

## Completion Criteria
3 FC430s and 3 FC830s provided via ESI to RH for the installation of OpenShift.

## Description
- [x] Create a year-long lease of 3 FC430s and 3 FC830s to RH for Openshift Beta cluster
- [ ] Confirm RH team member is able to access all 6 nodes for the cluster

## Completion dates
Desired - 2024-05-08
Required - TBD

**Comment by tzumainn** [comment]
Are there any restrictions for these nodes, or will any available do? Who/what project should be receiving these nodes?

**Comment by hpdempsey** [comment]
Please coordinate with @dystewart on the type of nodes.   He is currently using the existing test cluster for testing the new OpenShift releases and upgrade procedures, but we were tracking this cluster as a separate allocation because it was not clear that RHOAI and OpenShift upgrade testing could proceed all the way through the summer development and test phases on the right timeline to also be available for RHOI beta testing for other features.  The nodes should be as similar to production workers as our choices allow.

**Comment by tzumainn** [comment]
Okay! @dystewart just let me know if there are any other specifications, and let me know what the project name should be.

**Comment by joachimweyl** [comment]
@hpdempsey what is the priority of getting this up and running?

**Comment by hpdempsey** [comment]
I would like to have this cluster up and running by mid-August.   Project name should be OpenShiftBeta

**Comment by hpdempsey** [comment]
No further restrictions on the type of nodes as long as they are similar to production nodes (otherwise testing with the pre-release software on this cluster will not be very useful.

**Comment by hpdempsey** [comment]
@tssala23 may also be involved in this, so just adding him as a note

**Comment by tzumainn** [comment]
I've created the OpenShiftBeta project and added these nodes:

```
"MOC-R4PAC08U35-S1D","","OpenShiftBeta","fc430","available","False","","1ad955f1-5893-42c4-8926-b05ca5171f91"
"MOC-R4PAC08U35-S3A","","OpenShiftBeta","fc430","available","False","","96a0056a-a1bb-4a70-ac4c-dacba83173fb"
"MOC-R4PAC08U35-S3B","","OpenShiftBeta","fc430","available","False","","3bb7d3e6-241c-42ed-9404-bb1ccf8fe229"
"MOC-R4PAC10U13-S3","","OpenShiftBeta","fc830","available","False","","f69dbcba-4c04-4f74-9038-4ef9cccbabbb"
"MOC-R4PAC10U11-S1","","OpenShiftBeta","fc830","available","False","","1bc99a82-33fb-4c54-9db7-5e28d7a169d5"
"MOC-R4PAC10U11-S3","","OpenShiftBeta","fc830","available","False","","de114a30-e2c4-4c2f-b73e-48370dee33e1"
```

Both Dylan and Taj should be members of the project. Let me know if there are any issues!

**Comment by joachimweyl** [comment]
updated [NERC Additional Clusters](https://docs.google.com/spreadsheets/d/1QpqfGtMiNSTR48mQL-n54cojux1gjmbTyDmczekjfjk/edit?usp=sharing) sheet with new nodes.
